playground/src/playground.css

```diff
@@ -0,0 +1,6 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
```

Comment:
Is this `height: 100vh` actually needed? As far as I remember it is something the MC needs which somehow ended up in ui-kit, but our playground should not need it.

Reply:
not sure, I just added it so it looked the same as before 🤷‍♂️